.github/workflows/aws-deploy.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: Sample workflow to demonstrate AWS deployment including publishing data to S3 and invalidating a CloudFront distribution cache
+
+on: 
+  workflow_dispatch:
+    inputs:
+        environment:
+            description: "Environment to deploy to"
+            default: "staging"
+            options:
+                - staging
+                - production
+            required: true
+            type: choice
+       
+env:
+  # TODO: Verify what these values are for the allen-cell-animated Github organization
+  AWS_ACCOUNT_ID: ${{ vars.AWS_PUBLIC_DATA_RELEASES_ACCOUNT_ID }}
+  AWS_REGION: ${{ vars.AWS_DEFAULT_REGION }}
+  STAGING_CLOUDFRONT_DISTRIBUTION_ID: ${{ secrets.STAGING_CLOUDFRONT_DISTRIBUTION_ID }}
+  STAGING_S3_BUCKET: s3://staging-timelapse.allencell.org
```

Comment:
I think both cfe and simularium use the patterm `staging.appname.allencell.org` with a dot instead of hyphen.

Reply:
If that change requires another iteration or some discussion it need not hold up this PR

Reply:
Thanks, I ended up opening [`terraform-platform#1116`](https://aicsbitbucket.corp.alleninstitute.org/projects/SW/repos/terraform-platform/pull-requests/1116/overview) to change from `staging-timelapse.allencell.org` to `staging.timelapse.allencell.org`. Should be no big deal and now is a good time to get it right.